src/app/blog/[slug]/page.tsx

```diff
@@ -27,6 +27,7 @@ export function generateMetadata({ params }: BlogPostProps) {
   return createMetadata({
     seo: {
       ...data.seo,
+      title: data.title,
```

Comment:
>https://github.com/FilecoinFoundationWeb/filecoin-foundation/pull/753#pullrequestreview-2395676320

@CharlyMartin is right
```suggestion
      title: data.seo.title || data.title,
```